lib/vagrant-openshift/action/install_origin_asset_dependencies.rb

```diff
@@ -34,7 +34,11 @@ def call(env)
 ASSET_BACKUP_DIR=/data/asset_dependencies
 
 if ! which npm > /dev/null 2>&1 ; then
-  sudo yum -y --disablerepo=rhel-7-server-ose-3.1-rpms install npm
+  if [[ -e /etc/redhat-release && ! -e /etc/fedora-release && ! -e /etc/centos-release ]]; then
```

Comment:
CentOS and Fedora both have `/etc/redhat-release` so I don't think there's much reason to verify it exists.  I think we could simply leave the part that ensures `/etc/fedora-release` and `/etc/cenotos-release` are missing.